sample-app/src/components/VisualAutocomplete/SampleVisualSearchBar.tsx

```diff
@@ -12,6 +13,12 @@ export default function SampleVisualSearchBar() {
       screenReaderInstructionsId='SearchBar__srInstructions'
       headlessId='visual-autocomplete'
       entityPreviewsDebouncingTime={150}
+      verticalKeyToNameMapping={
```

Comment:
maybe we could have a `handleVerticalLink` prop instead? not sure if it's trying too hard to be generic, but I think it's a little more obvious what the prop is for.

Reply:
so `handleVerticalLink` would transform a given list of vertical keys to a VerticalLink object? it would work

Reply:
updated